examples/plugins/cmd-runner/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -x
+
+# This is the entrypoint for the image and meant to wrap the
+# logic of gathering/reporting results to the Sonobuoy worker.
+
+results_dir="${RESULTS_DIR:-/tmp/results}"
+
+if [[ $# -eq 0 ]]; then
+	echo "No arguments; expects each argument to be a shell command."
+	exit 1
+fi
+
+i=0
+while [ "$1" != "" ]; do
```

Comment:
Might be good to put some comments throughout this, just to be clear what each piece is doing for new users.